datasets/doc/source/how-to-use-with-numpy.rst

```diff
@@ -0,0 +1,60 @@
+Use with Numpy
+===================
+
+Let's integrate ``flwr-datasets`` with Numpy.
```

Comment:
Let's try to use the brand in the same way the framework/library itself does:
```suggestion
Use with NumPy
===================

Let's integrate ``flwr-datasets`` with NumPy.
```

Reply:
I fixed it also in the other places